View/ListItems/SongInfo.cs

```diff
@@ -33,5 +33,17 @@ public static List<SongInfo> ConvertSongListToSongInfo(Playlist playlist, List<S
 
             return returnList;
         }
+
+        public static List<SongInfo> ConvertSongListToSongInfo(List<Song> songs)
```

Comment:
Kunnen we niet gewoon PlaylistSong gebruiken ipv SongInfo?
Staat toch praktisch dezelfde info in?

Reply:
Nee, zit een paar verschillen in

Reply:
Zoals?

Reply:
PlaylistSong is voor een song in een playlist, en bij de search is dat niet van belang

Reply:
heeft maken met de duration van de song die anders niet geformat word via song